prow/git/types/types.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package git stores types used by all git clients
+package types
+
+// PullRequestMergeType enumerates the types of merges the GitHub API can
```

Comment:
is this still accurate? I thought the purpose of this refactoring is to also support gerrit specific merge types?

Reply:
Done